### ISSUE TITLE
Error in updateDatasetFolder: attempt to set 'colnames' on an object with less than two dimensions

### ISSUE THREAD

**Issue by mauromiguelm** [bigomics/omicsplayground#1014]
dataset: 730e1c337

This is a critical system maintained by @ivokwee so I am tagging you Ivo.

```r
[updateDatasetFolder] missing sigdb. Creating new sigdb file.
WARNING:: SD is constant. Skipping SD reduction...
calculating PCA 2D/3D...
Error in `colnames<-`(`*tmp*`, value = paste0("PC-", c("x", "y"))) : 
  attempt to set 'colnames' on an object with less than two dimensions
Calls: <Anonymous> ... pgx.clusterBigMatrix -> pgx.clusterMatrix -> colnames<-
```

**Comment by ivokwee** [comment]
dataset 730e1c337 computes successfully on Omics Playground v3.5.0-beta10+fix-1014240906

updateDatasetFolder finishes succesfully:
![image](https://github.com/user-attachments/assets/449dc81c-2491-4b90-b3ef-ee4396d568a1)


I suspect probable cause in clusterMatrix is matrix is constant or has NA. Perhaps when zero probes are not filtered out? I will test more.

**Comment by ivokwee** [comment]
I have tested pgx.clusterMatrix with matrix completely zero. Same "SD constant warning" but could not replicate crash in colnames. Closing this issue.

![image](https://github.com/user-attachments/assets/3aa0104f-773f-4b2a-a21b-1c33c4f6d69f)
